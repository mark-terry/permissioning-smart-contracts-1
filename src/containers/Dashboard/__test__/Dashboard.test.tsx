// Libs
import React from "react";
import toJson from "enzyme-to-json";
import { shallow, ShallowWrapper } from "enzyme";
// Components
import DashboardContainer from "../Dashboard";

describe("<Dashboard Container />", () => {
    let wrapper: ShallowWrapper;

    beforeEach(() => {
        jest.clearAllMocks();
        wrapper = shallow(<DashboardContainer />);
    });

    it("matches snapshot", () => {
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});
