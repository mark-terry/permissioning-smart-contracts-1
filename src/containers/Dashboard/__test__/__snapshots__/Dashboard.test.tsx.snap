// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Dashboard Container /> matches snapshot 1`] = `
<Memo(Dashboard)
  setTab={[Function]}
  tab="accountTab"
/>
`;
